### DISCUSSION TITLE
Problem: mqtt destination error

### DISCUSSION THREAD

**Discussion by Jambola77** [arendst/Tasmota#15680] [Q&A] [answered]
Hello,
I have modified my user_config_override.h as follows:
#undef  MQTT_HOST
#define MQTT_HOST         "192.168.1.100" 

#undef  MQTT_PORT
#define MQTT_PORT         1883

After flashing to a wemos mini, I changed the destination to 192.168.178.78 an I can see in the logs an error that 192.168.178.78 could not be reached at port 1883 (error -2). 
The 192.168.178.78 is in the same network and is a raspberry setup with plain mosquitto installation (sudo apt install mosquitto).
My Wemos has 192.168.178.70 and the network is working. Raspberry is also working and reachable.

Any idea, why mosquitto cannot be reached from wemos?

**Comment by sfromis** [comment, marked as answer]
Error -2 just means network error, without any suggestions about reason. If you really mean *plain Mosquitto installation* without any config, by default current Mosquitto versions will not allow network connections from outside the same machine, unless you configure to allow this.

For "most permissive" setup without security restrictions, I have this minimal `mosquitto.conf`:
```
listener 1883 0.0.0.0
allow_anonymous true
```

**Reply by Jambola77** [reply]
Thanks for this valuable hint! 
Allow_anonymous defaults to "false" which might be the reason for the behavior.

**Reply by sfromis** [reply]
In that case, I'd expect a security-related error code. Do check that you have opened Mosquitto to accept connections from the network, first line of the sample `mosquitto.conf`.
